labs/lab-1-html-and-css/desktop-stylesheet.css

```diff
@@ -0,0 +1,131 @@
+/*
+Author: Vikram Badarinath
+Created: 01/23/2018
+*/
+
+body {
+  color: #69696a;
+  background-color: #f9fafc;
+  font-family: Open Sans;
+  font-weight: 800;
+}
+
+/* top of the page*/
```

Comment:
Loving this commentary, though. And I love that you're thinking about organizing the file.